zerver/views/user_settings.py

```diff
@@ -36,26 +37,24 @@ def confirm_email_change(request, confirmation_key):
         raise JsonableError(_("Email address changes are disabled in this organization."))
 
     confirmation_key = confirmation_key.lower()
-    obj = get_object_from_key(confirmation_key)
-    confirmed = False
-    new_email = old_email = None  # type: Text
-    if obj:
-        confirmed = True
-        assert isinstance(obj, EmailChangeStatus)
-        new_email = obj.new_email
-        old_email = obj.old_email
-
-        do_change_user_email(obj.user_profile, obj.new_email)
-
-        context = {'realm': obj.realm,
-                   'new_email': new_email,
-                   }
-        send_email('zerver/emails/notify_change_in_email', to_email=old_email,
-                   from_name="Zulip Account Security", from_address=FromAddress.SUPPORT,
-                   context=context)
+    try:
+        obj = get_object_from_key(confirmation_key)
+    except ConfirmationKeyException as exception:
+        ctx = {'verbose_support_offers': settings.VERBOSE_SUPPORT_OFFERS}
```

Comment:
this can be removed (and possibly the ctx argument from `render_confirmation_key_error` can be removed entirely?)